README.md

```diff
@@ -16,11 +16,109 @@ var backlight = require('rpi-backlight');
 
 backlight.powerOn();
 backlight.powerOff();
+// 1 = Screen is OFF
+// 0 = Screen is ON
+backlight.getActualPowerValue();
+
+backlight.setMaxBrightness();
+backlight.getActualBrightness();
+// The screen goes Off at <= 9 brightness value
+backlight.setBrightnessValue(value);
+
+// Work in progress
+backlight.goToBrightnessValue(toValue, speed);
```

Comment:
If this is a work in progress, I'd just leave it out completely until it's ready, and add that in separate PR.